common/djangoapps/third_party_auth/strategy.py

```diff
@@ -34,3 +34,29 @@ def setting(self, name, default=None, backend=None):
         # At this point, we know 'name' is not set in a [OAuth2|LTI|SAML]ProviderConfig row.
         # It's probably a global Django setting like 'FIELDS_STORED_IN_SESSION':
         return super(ConfigurationModelStrategy, self).setting(name, default, backend)
+
+    def request_host(self):
+        """
+        Host in use for this request
+        """
+        # TODO: this override is a temporary measure until upstream python-social-auth patch is merged:
```

Comment:
@smarnach just a FYI - it merging PSA PR takes ages :) I haven't had any response on that PR yet, and there are 20+ PRs and 120+ issues pending on PSA. Looks like we're not going to have it any time soon.